Panic if local_path does not exist

**Describe the bug**
Panic if local_path does not exist

**To Reproduce**
Steps to reproduce the behaviour:
1. Set `local_path` to an unexistent path
2. Run
4. See error
```json
{"@timestamp":"2021-06-25T14:24:14Z","app":"s3sync-service","level":"error","message":"lstat /some/path: no such file or directory","version":"0.1.2"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x155ce38]

goroutine 103 [running]:
github.com/mazay/s3sync-service/service.FilePathWalkDir.func1(0xc00059c4a0, 0x1c, 0x0, 0x0, 0x1a99740, 0xc000c3e150, 0x184cd32, 0xb)
    /go/src/github.com/mazay/s3sync-service/service/local.go:58 +0x78
path/filepath.WalkDir(0xc00059c4a0, 0x1c, 0xc000365c88, 0xc000736080, 0xc00006e598)
    /usr/local/go/src/path/filepath/path.go:463 +0x8b
github.com/mazay/s3sync-service/service.FilePathWalkDir(0xc00014a320, 0x5, 0xc00059c4a0, 0x1c, 0xc0005f68d0, 0x17, 0xc00059c440, 0x19, 0xc00014a210, 0x9, ...)
    /go/src/github.com/mazay/s3sync-service/service/local.go:51 +0xd7
github.com/mazay/s3sync-service/service.syncSites.func1(0xc000148600, 0xc0005fa5a0, 0xc0005fa420, 0xc0005fa600, 0xc00014a320, 0x5, 0xc00059c4a0, 0x1c, 0xc0005f68d0, 0x17, ...)
    /go/src/github.com/mazay/s3sync-service/service/main.go:299 +0x294
created by github.com/mazay/s3sync-service/service.syncSites
    /go/src/github.com/mazay/s3sync-service/service/main.go:284 +0x105
```

**Expected behaviour**
There should be a proper error handler.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**
 - OS: Docker/k8s
 - Version 0.1.2

--- a/service/local.go
+++ b/service/local.go
@@ -58,6 +58,7 @@ func FilePathWalkDir(site *Site, awsItems map[string]string, uploadCh chan<- Upl
 			// Update errors metric
 			errorsMetric.WithLabelValues(site.LocalPath, site.Bucket, site.BucketPath, site.Name, "local").Inc()
 			logger.Error(err)
+			return err
 		}
 
 		if !d.IsDir() {

--- a/service/watcher.go
+++ b/service/watcher.go
@@ -63,7 +63,10 @@ func watch(site *Site, uploadCh chan<- UploadCFG,
 							if event.FileInfo.Mode().Type() == fs.ModeSymlink {
 								logger.Infof("%s is a symlink, skipping", event.Path)
 							} else {
-								fileWatcher(site, uploadCh, filepath)
+								err := fileWatcher(site, uploadCh, filepath)
+								if err != nil {
+									return
+								}
 							}
 						}
 					}
@@ -97,14 +100,18 @@ func watch(site *Site, uploadCh chan<- UploadCFG,
 
 // fileWatcher is watching for file mtime and adds the file into the upload queue if it's > 30 seconds old
 // Workaround for - https://github.com/radovskyb/watcher/issues/66
-func fileWatcher(site *Site, uploadCh chan<- UploadCFG, filepath string) {
+func fileWatcher(site *Site, uploadCh chan<- UploadCFG, filepath string) error {
 	for {
-		file, _ := os.Stat(filepath)
+		file, err := os.Stat(filepath)
+		if err != nil {
+			logger.Error(err)
+			return err
+		}
 		mtime := file.ModTime()
 		if time.Since(mtime).Seconds() >= 30 {
 			logger.Debugf("there were no writes to the file for 30 seconds, adding to the upload queue: %s", filepath)
 			uploadCh <- UploadCFG{filepath, site, "upload"}
-			return
+			return nil
 		}
 
 		logger.Debugf("looks like the file %s is still being written into, waiting", filepath)

